src/ios/CDVInAppBrowser.m

```diff
@@ -568,7 +576,11 @@ - (void)createViews
     self.spinner.userInteractionEnabled = NO;
     [self.spinner stopAnimating];
 
-    self.closeButton = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemDone target:self action:@selector(close)];
+    if (_browserOptions.hidenotclose != nil) {
```

Comment:
This shows a warning 
> Comparison between pointer and integer ('int' and 'void *')

Take a look how the other options are parsed and converted to booleans and do the same so you can use `if (browserOptions.hidenotclose)`